frontend/src/main/web/lib/stores/GlossaryStore.js

```diff
@@ -0,0 +1,522 @@
+import Dispatcher from '../dispatchers/GlossaryDispatcher';
+import assign from 'object-assign';
+import {EventEmitter} from 'events';
+import {Promise} from 'es6-promise';
+import Request from 'superagent';
+import Configs from '../constants/Configs';
+import {GlossaryActionTypes} from '../constants/ActionTypes';
+import GlossaryHelper from '../utils/GlossaryHelper'
+import StringUtils from '../utils/StringUtils'
+import DateHelpers from '../utils/DateHelper'
+import _ from 'lodash';
```

Comment:
I have started just importing the pieces of lodash I need, like `import { forEach, forOwn, isUndefined, isNull, set } from 'lodash'`. It makes the code look a little nicer, and will make it easy if we want to optimize later by only including a subset of lodash in our production build.